src/app/users/users-profile/users-profile.component.ts

```diff
@@ -38,4 +44,54 @@ export class UsersProfileComponent implements OnInit {
       console.log(error);
     });
   }
+
+  onSubmit(data, userDetail) {
+    const formdata = {
+      'password': data.password,
+      '_rev': userDetail._rev,
+      'name': userDetail.name,
+      'type': 'user',
+      'roles': userDetail.roles,
+      'firstName': userDetail.firstName,
+      'middleName': userDetail.middleName,
+      'lastName': userDetail.lastName,
+      'email': userDetail.email,
+      'language': userDetail.language,
+      'phoneNumber': userDetail.phoneNumber,
+      'birthDate': userDetail.birthDate,
+      'gender': userDetail.gender,
+      'level': userDetail.level,
+      '_attachments': userDetail._attachments
+    };
+    this.couchService.put(this.dbName + '/' + userDetail._id, formdata)
+      .subscribe((res) => {
+        this.couchService.delete('_session', { withCredentials: true }).subscribe((data: any) => {
```

Comment:
could we start a new session from the user credential as done on https://github.com/ole-vi/planet/pull/304